tests/README.md

```diff
@@ -20,9 +20,39 @@ This script creates `versionsTest` directory to which zstd repository is cloned.
 Then all tagged (released) versions of zstd are compiled.
 In the following step interoperability between zstd versions is checked.
 
+#### `automated-benchmarking.py` - script for benchmarking zstd prs to dev
+
+This script benchmarks facebook:dev and changes from pull requests made to zstd and compares
+them against facebook:dev to detect regressions. This script currently runs on a dedicated
+desktop machine for every pull request that is made to the zstd repo but can also
+be run on any machine via the command line interface.
+
+There are three modes of usage for this script: mode 0 will fetch all open pull requests
+in the repo and benchmark them against dev, mode 1 will fetch the pull request
```

Comment:
> mode 1 will fetch the pull request associated with the current hash and benchmark that against dev

What the scenario for this mode ?

Could it measure speed of the current hash against `dev` if it's different, irrespective of being associated to a PR or not ?

Reply:
This scenario was for being run in the CI tests. It finds the pr associated with the hash and then runs the benchmarking. The actual metrics that it gets might not be super meaningful since we don't know can't control the stability of the CI host machines but it might still be nice to have the output printed. 

If the hash isn't associated with a PR, it will crash. It would be trivial to have it compare the hash to facebook:dev regardless of association with a PR or not so I'll go ahead and add that feature in the update.